eng/common/scripts/Helpers/ApiView-Helpers.ps1

```diff
@@ -0,0 +1,64 @@
+function Map-LanguageToAPiview-Filter($language)
```

Comment:
This function name isn't very PowerShell.

Reply:
I would just remove the dashes :)